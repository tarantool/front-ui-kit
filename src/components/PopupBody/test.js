import React from 'react';
import renderer from 'react-test-renderer';
import { PopupBody } from './index';

jest.mock(
  '../Scrollbar/index.js',
  () => ({
    Scrollbar: ({ children, className }) => <div className={className}>{children}</div>
  })
);

describe('PopupBody', () => {
  it('renders without scrollbars', () => {
    const tree = renderer.create(
      <PopupBody className='external-class'>
        content
      </PopupBody>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders with scrollbars', () => {
    const tree = renderer.create(
      <PopupBody className='external-class' scrollable>
        content
      </PopupBody>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
