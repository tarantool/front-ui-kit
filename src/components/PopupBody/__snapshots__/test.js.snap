// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupBody renders with scrollbars 1`] = `
<div
  className="css-17w6qdz external-class"
>
  <div
    className="css-1wzifj2"
  >
    content
  </div>
</div>
`;

exports[`PopupBody renders without scrollbars 1`] = `
<div
  className="css-100xp3y external-class"
>
  content
</div>
`;
